src/AtulS/WeekDay.java

```diff
@@ -0,0 +1,53 @@
+//Write a Java program that keeps a number from the user and generates an integer between 1 and 7 
+//and displays the name of the weekday. (use switch case)
+package AtulS;
+
+import java.util.Scanner;
+
+public class WeekDay {
+
+	private void generateRandomNumber() { // It generate random number between
+											// 1-7. However it is generating
+											// number beyond 7 also. please
```

Comment:
Think is correct but here user give you number like 23 accordingly you have to populate day.